git_intro/git_intro.md

```diff
@@ -116,6 +125,37 @@ These modern systems also include powerful merging tools that make it possible f
 
 </div>
 
+## Git and GitHub
+
+Because [Git](https://git-scm.com/) is by far the most frequently used version control system in use today, it's worth explaining it in a bit more detail, and also disambiguating two terms that can be confusing: **Git** and **GitHub**.
+
+git
+---
+
+[Git](https://git-scm.com/) is a command line tool, a program that uses a set of rules that governs how the git version control system works. You can use it by itself and do everything you need to, if you’re comfortable working on the command line. Lots of people do just that, and that's actually the method we teach in our modules.
+
+GitHub
+---
+
+[GitHub](https://github.com) is a website (and ancillary software like GitHub Desktop that can work with the website or independently from it) that uses the git version control system and makes it easier to use and adds some helpful tools, particularly for use in teams, where more than one person needs to use git on the same set of files.
+
+While git by itself is great at version control and keeping track of your changes, GitHub wraps all of the sometimes complex inner workings of git into a visually pleasing, easy to understand user interface.
+
+If you're new to coding, feel overwhelmed by the idea of working on the command line, or just prefer a more visual interface, GitHub can be very helpful.
+
+<div class = "history">
+<b style="color: rgb(var(--color-highlight));">Historical context</b><br>
+
+Interestingly, despite the name, GitHub is independent from git, and is simply an organization that chose a name that was a portmanteau of "Git", the version control system it used, and "Hub", indicating a place where a lot of activity takes place.  They could do this because the word "git" was not trademarked.  Years after GitHub received **its** trademark, the originators of Git acted to keep this from happening again by finally trademarking the word "git".  They now protect that trademark and have acted to prevent similar portmanteaus.  There doesn't seem to be any ill will between the organizations, but it's certainly an interesting history!
```

Comment:
```suggestion
Interestingly, despite the name, GitHub is independent from git, and is simply an organization that chose a name that was a portmanteau of "Git," the version control system it used, and "Hub," indicating a place where a lot of activity takes place.  They could do this because the word "git" was not trademarked.  Years after GitHub received **its** trademark, the originators of Git acted to keep this from happening again by finally trademarking the word "git."  They now protect that trademark and have acted to prevent similar portmanteaus.  There doesn't seem to be any ill will between the organizations, but it's certainly an interesting history!
```